seacrowd/utils/schemas/chat.py

```diff
@@ -0,0 +1,24 @@
+"""
+Conversational Chat Schema
+"""
+import datasets
+
+features = datasets.Features(
+    {
+        "id": datasets.Value("string"),
+        "input": datasets.Sequence({
+            "role": datasets.ClassLabel(names=["system", "user", "assistant"]),
```

Comment:
hi @SamuelCahyawijaya @yongzx just let you know the changes on schema has been merged to master, but with this `role` field being changed to `string` (`datasets.Value("string")`) due to possibilities of additional/custom roles and HF mechanics that return an indices of the label for their examples had it been set as ClassLabel (which is less intuitive than string)